server/src/main/java/com/linecorp/centraldogma/server/metadata/MetadataService.java

```diff
@@ -322,6 +322,22 @@ public MetadataService(ProjectManager projectManager, CommandExecutor executor)
         return metadataRepo.push(projectName, Project.REPO_DOGMA, author, commitSummary, change);
     }
 
+    /**
+     * Purge a {@link RepositoryMetadata} of the specified {@code repoName} from the specified
+     * {@code projectName}.
+     */
+    public CompletableFuture<Revision> purgeRepo(Author author, String projectName, String repoName) {
+        requireNonNull(author, "author");
+        requireNonNull(projectName, "projectName");
+        requireNonNull(repoName, "repoName");
+
+        final JsonPointer path = JsonPointer.compile("/repos" + encodeSegment(repoName));
+        final Change<JsonNode> change = Change.ofJsonPatch(METADATA_JSON,
+                                                           new RemoveOperation(path).toJsonNode());
+        final String commitSummary = "Purge the repo '" + repoName + "' from the project " + projectName;
```

Comment:
nit: Could we add quotes for project names as well throughout this class?

Reply:
Sure! 😀